ooni/errors.py

```diff
@@ -148,6 +149,9 @@ def failureToString(failure):
     elif isinstance(failure.value, ProcessDone):
         string = 'process_done'
 
+    elif isinstance(failure.value, ConnectionDone):
+        string = 'connection_done'
```

Comment:
Re. proxy_connection_eof, Twisted returns ConnectionDone, but we don't know whether it's unexpected EOF or not.
I guess we could get ConnectionDone under other circumstances, don't you think?